app/controllers/UserTokenController.scala

```diff
@@ -43,25 +43,27 @@ class UserTokenController @Inject()(dataSetDAO: DataSetDAO,
     }
   }
 
-  def validateUserAccess(name: String, token: String) = wkDataStoreActions.DataStoreAction(name).async(validateJson[UserAccessRequest]) { implicit request =>
-    val accessRequest = request.body
-    if (token == DataStoreHandler.webKnossosToken) {
-      Fox.successful(Ok(Json.toJson(UserAccessAnswer(true))))
-    } else {
-      for {
-        userBox <- bearerTokenService.userForToken(token)(GlobalAccessContext).futureBox
-        ctxFromUserBox = DBAccessContext(userBox)
-        ctx = URLSharing.fallbackTokenAccessContext(Some(token))(ctxFromUserBox)
-        answer <- accessRequest.resourceType match {
-          case AccessResourceType.datasource =>
-            handleDataSourceAccess(accessRequest.resourceId, accessRequest.mode, userBox)(ctx)
-          case AccessResourceType.tracing =>
-            handleTracingAccess(accessRequest.resourceId, accessRequest.mode, userBox)(ctx)
-          case _ =>
-            Fox.successful(UserAccessAnswer(false, Some("Invalid access token.")))
+  def validateUserAccess(name: String, token: String) = Action.async(validateJson[UserAccessRequest]) { implicit request =>
```

Comment:
ActionBuilder cannot take parameters anymore (or at least I couldn’t figure out how) so I changed the `DataStoreAction`s to `Action`s with `dataStoreService.validateAccess` blocks. Same below in `WKDataStoreController`